app/src/main/java/com/vestrel00/daggerbutterknifemvp/ui/common/BaseActivity.java

```diff
@@ -46,6 +46,12 @@
     @Inject
     protected Navigator navigator;
 
+    /**
+     * A reference to the FragmentManager is injected and used instead of the getter method. This
+     * enables ease of mocking and verification in tests (in case Fragment needs testing).
```

Comment:
"(in case Fragment needs testing)" -> "(in case Activity needs testing)". Fixed in https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/3e475aaa3d9a11309b740761ede1fd31483d1491